nx-monorepo/README.md

```diff
@@ -1,34 +1,32 @@
 # Nx Monorepo
 
-This example shows how to use Nx to organize a mono repo and track dependencies.
+This example shows how to use Nx to organize a mono repo and track dependencies between the packages in the monorepo.
 
 The example consists of the following components:
 
-```
- - packages:
-     - s3folder: ComponentResource that manages a S3 bucket
-     - website-deploy: ComponentResource resource that manages files in a S3 bucket
-     - website-builder: Mock website generator that creates HTML output
- - infra: Pulumi program that uses the s3folder and website ComponentResources to deploy a website
-```
+- **components/s3folder**: A [Component Resource](https://www.pulumi.com/docs/concepts/resources/components/) that manages a S3 bucket and its access policies.
+- **components/website-deploy**: A [Component Resource](https://www.pulumi.com/docs/concepts/resources/components/) resource that manages files in a S3 bucket
```

Comment:
This reads as `A Component Resource resource that...` the word resource doesn't need to be in the sentence twice